code/modules/mob/mob_defines.dm

```diff
@@ -76,14 +76,17 @@
 	var/m_intent = "run"//Living
 	var/lastKnownIP = null
 	var/atom/movable/buckled = null//Living
+	var/obj/item/l_hand = null//Living
+	var/obj/item/r_hand = null//Living
+	var/obj/item/weapon/back = null//Human/Monkey
```

Comment:
Bonus; this will conflict with #20714 which actually already does what Remie is telling you to do.